cli/cli-client/src/main/java/org/infinispan/cli/logging/Messages.java

```diff
@@ -92,4 +95,64 @@
 
    @Message("Invalid resource '%s'")
    IllegalArgumentException invalidResource(String name);
+
+   @Message("No patches installed")
+   String patchNoPatchesInstalled();
+
+   @Message("%s")
+   String patchInfo(PatchInfo patchInfo);
+
+   @Message("The supplied patch cannot be applied to %s %s")
+   IllegalStateException patchCannotApply(String brandName, String version);
+
+   @Message("File %s SHA mismatch. Expected = %s, Actual = %s")
+   String patchShaMismatch(Path path, String digest, String sha256);
+
+   @Message("The following errors were encountered while validating local installation:")
+   String patchValidationErrors();
```

Comment:
print the errors? `org.infinispan.cli.patching.PatchTool#installPatch()` isn't printing the `error` list.

Reply:
Fixed